### DISCUSSION TITLE
How much longer will AntD v4 be supported?

### DISCUSSION THREAD

**Discussion by kiwikencox** [ant-design/ant-design#45623] [General]
I just want to confirm the support date for V4 as the [V5 release post on Medium](https://medium.com/ant-design/ant-design-5-0-is-released-27b2552f14a3) says its the end of 2023.

Will v4 continue to receive security updates etc going forward?

**Comment by zombieJ** [comment]
antd not using magic code for the feature. It all follow the React life cycle. But if meet the critical security issue, I guess the answer is yes.
